lib/PatchNavigation.mli

```diff
@@ -0,0 +1,24 @@
+(** This module defines navigation and summary functions for viewing changes in a diff viewer application using the Nottui library. It supports navigating between patches and displaying summaries of changes. *)
+
+type direction =
+  | Prev
+  | Next
+      (** Type representing navigation directions within a list of patches. 
+          - [Prev] indicates navigation to the previous item.
+          - [Next] indicates navigation to the next item. *)
+
+val quit : bool Lwd.var
+(** [quit] is a variable indicating whether the application should terminate its main loop. *)
```

Comment:
Shouldn't this be in InteractiveViewer?

It seems to be unrelated to patch navigation to me!